client/blocks/reader-related-card-v2/style.scss

```diff
@@ -442,3 +442,40 @@ $reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
 		max-height: none;
 	}
 }
+
+// Placeholders
+.reader-related-card-v2.is-placeholder {
```

Comment:
@bluefuton Actually nvm. It doesn't make sense in this case. I tried and we're going to end up with too many `.is-modifers`.